package.json

```diff
@@ -10,7 +10,8 @@
     "prebuild": "rm -rf dist",
     "prepack": "npm run build",
     "pretest": "npm run lint && npm run prettier",
-    "prettier": "prettier --check --write \"./**/**.{js,ts,md,json}\"",
+    "prettier": "prettier --check .",
+    "prettier:write": "prettier --check --write .",
```

Comment:
I like the separation better too!

What do you think about adding one for `lint` too, but consolidating the name to `...:fix`?
```
"lint:fix": "eslint . --ext .js,.ts --fix",
...
"prettier:fix": "prettier --check --write .",
```

Reply:
great call, added in https://github.com/readmeio/http-status-codes/pull/73/commits/3aed6cca056d49f5380cd688416d26b1ad273576!